### ISSUE TITLE
Update text-prompts

### ISSUE THREAD

**Issue by sirius-co** [zqhang/AnomalyCLIP#46]
Hi,

thank you for sharing your implementation!
I was wondering if changing the prompts in prompt_ensemble.py will impact somehow the performance, but I am not sure if it does. Let's say I change prompt_templates in prompt_ensemble.py, how is it supposed to impact the AnomalyCLIP performance? 

Thank you!

**Comment by zqhang** [comment]
Yes, it will impact the performance. An appropriate design may further improve its performance.

**Comment by sirius-co** [comment]
Thank you for your answer! 
Please correct me if I am wrong, you do not use at all these prompts from prompt_templates, right? If so, then how can the prompts_template update impact the performance of AnomalyCLIP?
